### DISCUSSION TITLE
Disable Discharge

### DISCUSSION THREAD

**Discussion by CriticalError666** [mtrossbach/noah-mqtt#15] [General]
Hello Markus,

thank you for your work and the great project.
Do you think there will be a way to disable the discharge in the future? 

I'm currently trying to integrate my battery into my zero export project and only use the storage from 8 p.m. If the system provides not enough Power after 6:30 p.m., it will be pulled from the NOAH before 8 p.m. That's why I'm looking for a way to stop the discharche

Best regards,
Stefan

**Comment by mtrossbach** [comment]
Hi Stefan,

Thank you for your feedback and for using the project!

One potential solution you could try is setting the system output power to 0W. This should stop the discharge, as the system won't output any power. However, please note that if you've set a schedule (like a work mode), it will override the system output power settings, so the system will follow the schedule instead.

I'm not entirely sure if this will work, but it probably should. I'll try to test it tomorrow, as I wasn't able to fully charge the battery today due to bad weather.

I hope this helps with your zero export project!

Best regards,  
Markus

**Reply by msmthng** [reply]
For me it works to set 0W on the Inverter (hm with AhoyDTU). (most of the time, sometimes noah does some strenge things)

Btw thanks for the nice project, i was thinking to do something similar but then i found your nice work!

**Reply by MartinGoX** [reply]
Hi all.
I've set the system output power to zero but it will not work in all cases. I've 2 work modes:
1. between 8am and 6pm 'Battery first'
2. between 8pm and 6am 'Load first'
So there is a gap between 6pm and 8pm
The output power during mode 2 is as expected and after the battery is empty the output is zero.
In mode 2 after the battery is fully charged the output power remains at 50 Watt lowest (is more if enough solar power is available). These 50W remains until phase 1 begins.
I do not know if it belongs to this 'gap' phase, maybe I have to extend phase 1 up to 8pm and check it again.

Of course, setting inverter output to zero should work, but I have a NEO800MX which cannot be controlled by HA - or does someone have a solution for that?

Best regards
Martin

**Reply by frankfrommelt** [reply]
Hi,
I recently discovered there is a direct integration with Shelly (in my case Shelly 3EM) and Noah2000 available.
So in the Shine Phone app you can now have the Shelly device control to output of your Noah2000 according to the current consumption in your home.
So, only the amount of energy that you are "using" in your home is fed into the AC wiring.
Maybe this helps
   Frank
